<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PositiveObstacleDialog</class>
 <widget class="QDialog" name="PositiveObstacleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>216</width>
    <height>181</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>216</width>
    <height>181</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>216</width>
    <height>181</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="minimumSize">
          <size>
           <width>44</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>X:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="dPOX"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="minimumSize">
          <size>
           <width>44</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Y:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="dPOY"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="minimumSize">
          <size>
           <width>44</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Width</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="dPOWidth"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="minimumSize">
          <size>
           <width>44</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Height:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="dPOHeight"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="dApplyBtn">
         <property name="text">
          <string>Apply</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="dCancelBtn">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
